src/main/java/io/vinyldns/java/model/membership/MemberId.java

```diff
@@ -0,0 +1,33 @@
+package io.vinyldns.java.model.membership;
+
+public class MemberId {
```

Comment:
`MemberId` is required because the create group endpoint expects a collection of `{'id': <id>}` values, not simply a collection of ids.